tests/lib/rules/no-unused-prop-types.js

```diff
@@ -3148,24 +3148,7 @@ ruleTester.run('no-unused-prop-types', rule, {
         line: 11,
         column: 8
       }]
-    }, { // None of the props are used
-      code: [
-        'var Hello = createReactClass({',
```

Comment:
In this case, the duplicated test isn't a duplicate; one imports React, and the other doesn't, and it's intentional to check both.